controllers/discovery/initDiscoveryUserData.js

```diff
@@ -69,11 +75,21 @@ const InitDiscoveryUserData = async (req, res) => {
     });
     let suggestedUsers = usersWithCommonFollowerResult;
 
+    let totalData = await sequelize.query(totalDataQuery, {
+      type: QueryTypes.SELECT,
+      replacements: {
+        userId
+      },
+      raw: true
+    });
+    totalData = totalData?.[0]?.total_data || 0;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The optional chaining operator (`?.`) is used here, which is great for preventing runtime errors when trying to access properties of `undefined` or `null`. However, it's worth noting that if `totalData` is `null` or `undefined`, it will default to `0`. Make sure this behavior is expected and handled appropriately in your application logic.



<!-- This is an auto-generated comment by OSS CodeRabbit -->